src/api/entrypoints/alunos/views.py

```diff
@@ -83,3 +82,17 @@ async def get_aluno_email(aluno_email: str, repository=Depends(get_repo())):
 @router.get("/orientador/{orientador_id}", response_model=List[AlunoInDB])
 async def get_alunos_por_orientador(orientador_id: int, repository=Depends(get_repo())):
     return await ServicoAluno(repository).buscar_alunos_por_orientador(orientador_id)
+
+@router.put("/remover-orientador/{aluno_id}", response_model=AlunoInDB)
+async def remover_orientador_aluno(aluno_id: int,token: str = Depends(oauth2_scheme), repository=Depends(get_repo())):
+    logger.info(f"Solicitado deleção de {aluno_id=} | Autenticando usuário atual.")
```

Comment:
Corrigir log para o que está realmente sendo realizado nesse fluxo.
```suggestion
    logger.info(f"Solicitado remoção do orientador do {aluno_id=} | Autenticando usuário atual.")
```

Reply:
Obrigado, ajustei para a próxima atualização.